app/scripts/lib/mascara-detection-usage.js

```diff
@@ -0,0 +1,69 @@
+module.exports = MascaraDetectionUsageSetup
+
+function MascaraDetectionUsageSetup (provider) {
+  const self = this
+
+  const head = document.getElementsByTagName('head')[0]
+  const headContent = head.children
+  const childrenLength = headContent.length
+  // the default is to serve the extensions provider
+  self.mascara = false
+  self.extensionProvider = provider
+  for (let index = 0; index < childrenLength; index++) {
+    const src = headContent[index].getAttribute('src')
+    if (src === 'MASCARA_PROXY_ORIGIN') {
```

Comment:
lets make this a CONSTANT_VARIABLE and then set it at the top or somewhere

we could also prolly do this via process.env.ENV_VAR